### PR TITLE
added book impl and implemented service and repo levels

### PR DESCRIPTION
**Pull request by pavlov-n**

### CODE REVIEW

**Review by sonyasapsan — APPROVED** [review]

File: src/main/java/com/bookshop/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```
Comment (liliia-ponomarenko) [inline src/main/java/com/bookshop/model/Book.java]: Lombok is a great library but be [attentive when using it with Entity](https://jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/) JFYI

File: src/main/java/com/bookshop/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    @Column(nullable = false, unique = true)
+    private String isbn;
+
+    @Column(nullable = false)
+    private BigDecimal price;
+
+    private String description;
+
+    private String coverImage;
```
Comment (liliia-ponomarenko) [inline src/main/java/com/bookshop/model/Book.java]:
```suggestion
    private String description;
    private String coverImage;
```

File: src/main/java/com/bookshop/service/impl/BookServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import com.bookshop.service.BookService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+    private BookRepository bookRepository;
+
+    @Autowired
```
Comment (liliia-ponomarenko) [inline src/main/java/com/bookshop/service/impl/BookServiceImpl.java]:
```suggestion
```
This annotation is redundant (more details in spring documentation)

**Review by liliia-ponomarenko — APPROVED** [review]
Good job! But fix minor mistakes before merging ;)

**Pull request merged**